pkg/controller/activegate/activegate_update_test.go

```diff
@@ -21,33 +22,183 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
+const (
+	mockActivegateVersion = "1.200.0"
+)
+
 func init() {
 	_ = apis.AddToScheme(scheme.Scheme) // Register OneAgent and Istio object schemas.
 	_ = os.Setenv(k8sutil.WatchNamespaceEnvVar, _const.DynatraceNamespace)
 }
 
+type mockIsLatestUpdateService struct{}
```

Comment:
Alternatively you can use a mocking framework, we use one on `dtclient/mock_client.go`

Reply:
I don't mind leaving it for follow-up change.